beginner_source/transfer_learning_tutorial.py

```diff
@@ -253,7 +253,7 @@ def visualize_model(model, num_images=6):
             images_so_far += 1
             ax = plt.subplot(num_images//2, 2, images_so_far)
             ax.axis('off')
-            ax.set_title('predicted: {}'.format(dset_classes[labels.data[j]]))
+            ax.set_title('predicted: {}, truth {}'.format(dset_classes[preds.numpy()[j][0]], dset_classes[labels.data[j]]))
```

Comment:
Thanks, 
But, there's not enough space to show both truth and predicted.
Can you please use:
`ax.set_title('predicted: {}'.format(dset_classes[preds[j,0]]))`